api/openapi-spec/swagger.json

```diff
@@ -2,7 +2,7 @@
   "swagger": "2.0",
   "info": {
    "title": "Kubernetes",
-   "version": "v1.12.0"
```

Comment:
We don't need this.

I think this will disappear after these steps:
```
git checkout master
git pull upstream master --tags
git checkout xxx
git rebase master
```